src/python/pants/backend/visibility/glob.py

```diff
@@ -25,6 +26,12 @@ class PathGlobAnchorMode(Enum):
     def parse(cls, pattern: str) -> PathGlobAnchorMode:
         for mode in cls.__members__.values():
             if pattern.startswith(mode.value):
+                if mode is PathGlobAnchorMode.INVOKED_PATH:
+                    # Special case invoked path, to not select ".text"; only "." "../" or "./" are
+                    # valid for `invoked path` mode. (we're not picky on the number of leading dots)
+                    # Using `not in "/"` here rather than `!= "/"` as it also works for `""`.
```

Comment:
Instead of using a non-obvious trick and then having to document it, it might be better to just make the code more obvious? E.g., 

`if pattern.lstrip(".")[:1] not in ["/", ""]:`

Reply:
well now I feel that comment backfired 🤣 as I went in and added it as an after thought, but you're right, could just use the more obvious version you suggest :)

Reply:
(worth the CI cycle to fix it?)